certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2934 @@
+            "x-ms-visibility": "internal",
+            "schema": {
+              "type": "object"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Existing projects returned",
+            "schema": {
+              "$ref": "#/definitions/ProjectResponseListResponse"
+            },
+            "examples": {
+              "application/json": {
+                "data": [
+                  {
+                    "id": "project-uuid-1",
+                    "attributes": {
+                      "Name": "Sample Project1",
+                      "Manager": [
+                        {
+                          "id": "manager-uuid",
+                          "fullName": "Person 1",
+                          "imageId": null
+                        }
+                      ],
+                      "Program": [
+                        {
+                          "id": "program-uuid",
+                          "name": "Program 1"
+                        }
+                      ],
+                      "Portfolio": [
+                        {
+                          "id": "portfolio-uuid",
+                          "name": "Portfolio 1"
+                        }
+                      ],
+                      "Custom Field1": 123
+                    }
+                  },
+                  {
+                    "id": "project-uuid-2",
+                    "attributes": {
+                      "Name": "Sample Project2",
+                      "Custom Field1": 345,
+                      "Custom Field2": "sample value"
+                    }
+                  }
+                ]
+              }
+            }
+          },
+          "401": {
+            "description": "User not authorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "403": {
+            "description": "Operation not allowed",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Projects"
+        ],
+        "summary": "Create a Project",
+        "description": "Creates a new Project",
+        "operationId": "ProjectsCreate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/CreateProjectRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "New project returned",
+            "schema": {
+              "$ref": "#/definitions/ProjectResponseResponse"
+            },
+            "examples": {
+              "application/json": {
+                "data": {
+                  "id": "project-uuid-",

```

Comment:
@vsorokin-ppmexpress a small typo here, please make it `project-uuid` so that it looks complete.